src/org/scada_lts/dao/migration/mysql/V2_3__FaultsAndAlarms.java

```diff
@@ -30,70 +35,68 @@
 
 /**
  *
- * @author kamil.jarmusik@gmail.com
+ * @author kamil.jarmusik@gmail.com, grzegorz.bylica@gmail.com
  *
  */
 
-public class V2_3__FaultsAndAlarms implements SpringJdbcMigration {
+public class V2_3__FaultsAndAlarms extends BaseJavaMigration {
 
-    public void migrate(JdbcTemplate jdbcTmp) throws Exception {
+    private static final Log LOG = LogFactory.getLog(V2_3__FaultsAndAlarms.class);
 
-        try {
-            addColumnsToDataPointsTable(jdbcTmp);
-            updateDataPointsTable(jdbcTmp);
-            createPlcAlarmsTable(jdbcTmp);
-            createFunctions(jdbcTmp);
-            createViews(jdbcTmp);
-            createProcedure(jdbcTmp);
-            createTrigger(jdbcTmp);
-        } catch (Exception ex) {
```

Comment:
Thanks to the old solution, we had to log every jdbc error in the console, we had a stack, so we could quickly locate the problematic place in the code, and find out the exact exception. In the current solution, we will only get information about a wrong query, it will be more difficult to find where the exception appeared in the code.